app/admin/rents.rb

```diff
@@ -0,0 +1,27 @@
+ActiveAdmin.register Rent do
+  permit_params :user, :book, :from, :to
+
+  index do
+    selectable_column
+    id_column
+    column :user
+    column :book
+    column :from
+    column :to
+  end
+
+  filter :user
+  filter :book
+  filter :from
+  filter :to
+
+  form do |f|
+    f.inputs do
+      f.input :user, as: :select, collection: User.all.collect { |user| [user.email] }
```

Comment:
I think this should be `f.input :user_id, as: :select, collection: User.all.collect { |user| [user.id, user.email] }`, check it and let me know!